pkg/app/src/components/Account/TempData/index.ts

```diff
@@ -0,0 +1,297 @@
+export const tempBalances = [
```

Comment:
mock data should be provided through graphql so you do not need to modify the component later.

Reply:
do you have a sample or example for that?